core/arch/arm/tee/pta_generic.c

```diff
@@ -0,0 +1,287 @@
+// SPDX-License-Identifier: BSD-2-Clause
```

Comment:
`pta_socket.c` socket was put in `core/arch/arm/tee/` since it's part of the implementation of the GP interface used by the TAs. Since then we've decided to put everything in `core/arch/arm/pta/` mixing production code with test code. A bit messy, if you ask me.

Reply:
Right. Let's continue the discussion on https://github.com/OP-TEE/optee_os/pull/3274.